frontend/src/container/ListAlertRules/AlertsEmptyState/AlertsEmptyState.tsx

```diff
@@ -0,0 +1,233 @@
+import AppReducer from 'types/reducer/app';
+
+function InfoLinkText({
+	infoText,
+	link,
+	leftIconVisible,
+	rightIconVisible,
+}: {
+	infoText: string;
+	link: string;
+	leftIconVisible: boolean;
+	rightIconVisible: boolean;
+}): JSX.Element {
+	return (
+		<Flex
+			onClick={() => window.open(link, '_blank')}
+			className="info-link-container"
+		>
+			{leftIconVisible && <PlayCircleFilled />}
+			<Typography.Text className="info-text">{infoText}</Typography.Text>
+			{rightIconVisible && <ArrowRightOutlined rotate={315} />}
+		</Flex>
+	);
+}
+
+export function AlertsEmptyState(): JSX.Element {
+	const { t } = useTranslation('common');
+	const { role, featureResponse } = useSelector<AppState, AppReducer>(
+		(state) => state.app,
+	);
+	const [addNewAlert] = useComponentPermission(
+		['add_new_alert', 'action'],
+		role,
+	);
+
+	const { notifications: notificationsApi } = useNotifications();
+
+	const handleError = useCallback((): void => {
+		notificationsApi.error({
+			message: t('something_went_wrong'),
+		});
+	}, [notificationsApi, t]);
+
+	const onClickNewAlertHandler = useCallback(() => {
+		featureResponse
+			.refetch()
+			.then(() => {
+				history.push(ROUTES.ALERTS_NEW);
+			})
+			.catch(handleError);
+	}, [featureResponse, handleError]);
+
+	return (
+		<div className="alert-list-container">
+			<div className="alert-list-view-content">
+				<div className="alert-list-title-container">
+					<Typography.Title className="title">Alert Rules</Typography.Title>
+					<Typography.Text className="subtitle">
+						Create and manage alert rules for your resources.
+					</Typography.Text>
+				</div>
+				<section className="empty-alert-info-container">
+					<div className="alert-content">
+						<section className="heading">
+							<img
+								src="/Icons/alert_emoji.svg"
+								alt="header-image"
+								style={{ height: '32px', width: '32px' }}
+							/>
+							<div>
+								<Typography.Text className="empty-info">
+									No Alert rules yet.{' '}
+								</Typography.Text>
+								<Typography.Text className="empty-alert-action">
+									Create an Alert Rule to get started
+								</Typography.Text>
+							</div>
+						</section>
+						<div className="action-container">
+							<Button
+								className="add-alert-btn"
+								onClick={onClickNewAlertHandler}
+								icon={<PlusOutlined />}
+								disabled={!addNewAlert}
+								type="primary"
+								data-testid="add-alert"
+							>
+								New Alert Rule
+							</Button>
+							{InfoLinkText({
+								infoText: 'Watch a tutorial on creating a sample alert',
+								link: 'https://youtu.be/xjxNIqiv4_M',
+								leftIconVisible: true,
+								rightIconVisible: true,
+							})}
+						</div>
+
+						{InfoLinkText({
+							infoText: 'How to create Metrics-based alerts',
+							link:
+								'https://signoz.io/docs/alerts-management/metrics-based-alerts/?utm_source=product&utm_medium=alert-empty-page',
```

Comment:
Maintain constants so that it's easy to read.

Reply:
It's all for the infoText and Links

Reply:
done moved to separate util and files